tests/Feature/Availability/AvailabilityCreateTest.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+namespace Tests\Feature\Availability;
+
+use App\Modules\Availability\Availability;
+use App\Modules\Endorsement\Special\Assignment;
+use App\Modules\Endorsement\Special\SpecialEndorsement;
+use App\Modules\Position\Position;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Tests\TestCase;
+use Tests\Unit\Availability\AvailabilityTestHelper;
+
+class AvailabilityCreateTest extends TestCase
+{
+    use RefreshDatabase, MakesGraphQLRequests, AvailabilityTestHelper;
+
+    protected $position;
+    public $expectedExceptions = [
+        'errors' => [
+            [
+                'message',
+                'extensions',
+                'locations',
+            ],
+        ],
+    ];
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->actingAs($this->mockedUser());
+        $this->mockUserFind();
+
+        $this->position = factory(Position::class)->create();
```

Comment:
Lots of bits and bobs from the original booking test lying around here (such as this and comments). Can we clear these up?

Reply:
done